schemas/treatment.json

```diff
@@ -178,6 +178,30 @@
         "core": true,
         "displayName": "Response to Therapy"
       }
+    },
+    {
+      "name": "clinical_trials_database",
+      "description": "",
```

Comment:
Suggested description: 

If the donor is a participant in a clinical trial, indicate the clinical trial database where the clinical trial is registered.

Reply:
Thanks @rosibaj I mistakenly left out the descriptions!